packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -1,79 +1,39 @@
 <script lang="typescript">
-    import { get } from 'svelte/store'
     import { onDestroy, onMount } from 'svelte'
     import { Animation, Button, Icon, OnboardingLayout, Text } from 'shared/components'
-    import { mobile } from 'shared/lib/app'
-    import { convertToFiat, currencies, exchangeRates, formatCurrency } from 'shared/lib/currency'
     import { Platform } from 'shared/lib/platform'
     import { promptUserToConnectLedger } from 'shared/lib/ledger'
-    import {
-        LOG_FILE_NAME,
-        migration,
-        migrationLog,
-        resetMigrationState,
-        totalMigratedBalance,
-    } from 'shared/lib/migration'
-    import {
-        activeProfile,
-        newProfile,
-        profileInProgress,
-        saveProfile,
-        setActiveProfile,
-        updateProfile,
-    } from 'shared/lib/profile'
-    import { appRouter, ledgerRouter } from '@core/router'
-    import { LedgerAppName } from 'shared/lib/typings/ledger'
-    import { formatUnitBestMatch } from 'shared/lib/units'
-    import { getProfileDataPath, walletSetupType } from 'shared/lib/wallet'
-    import { AvailableExchangeRates, CurrencyTypes } from 'shared/lib/typings/currency'
-    import { Locale } from '@core/i18n'
+    import { LOG_FILE_NAME, migration, migrationLog, resetMigrationState } from 'shared/lib/migration'
+    import { activeProfile, updateProfile } from 'shared/lib/profile'
+    import { AppRoute, appRouter, ledgerRouter } from '@core/router'
+    import { api, getProfileDataPath, walletSetupType } from 'shared/lib/wallet'
+    import { Locale, localize } from '@core/i18n'
     import { SetupType } from 'shared/lib/typings/setup'
+    import { showAppNotification } from '@lib/notifications'
+    import { openPopup } from '@lib/popup'
+    import { getDefaultStrongholdName } from '@lib/utils'
 
     export let locale: Locale
 
     const { didComplete } = $migration
-
     const wasMigrated = $didComplete
 
-    let localizedBody = 'body'
-    let localizedValues = {}
     let logExported = false
+    let isNotALedgerProfile: boolean = false
+    let exportStrongholdBusy = false
```

Comment:
how we set this to true and to false i dont think is accurate, it should be set to true right after clicking the export button, and set to false when there is an error or the export finishes